quickwit-metastore/src/metastore/mod.rs

```diff
@@ -188,4 +211,38 @@ pub trait Metastore: Send + Sync + 'static {
 
     /// Returns the metastore uri.
     fn uri(&self) -> &Uri;
+
+    /// Gets the last delete opstamp for a given `index_id`.
+    async fn last_delete_opstamp(&self, index_id: &str) -> MetastoreResult<u64>;
+
+    /// Creates a [`DeleteTask`] from a [`DeleteQuery`].
+    async fn create_delete_task(&self, delete_query: DeleteQuery) -> MetastoreResult<DeleteTask>;
+
+    /// Deletes all [`DeleteTask`] for a given `index_id`.
+    async fn delete_delete_tasks(&self, index_id: &str) -> MetastoreResult<()>;
```

Comment:
why do we have this call? It is not used as far as I can tell.

I don't think this is a good idea.

Reply:
(also the name is silly)

Reply:
Agreed, I used it only for testing so it's useless.